app/controllers/works_controller.rb

```diff
@@ -82,7 +82,7 @@ def show
 
   def file_list
     @work = Work.find(params[:id])
-    render json: @work.pre_curation_uploads_fast
+    render json: @work.uploads
```

Comment:
`work.uploads()` in the model handles the logic of returning pre/post curated files.